neuron/src/app/Neuron/Reader/Org.hs

```diff
@@ -36,16 +36,22 @@ parseOrg _ s = do
 extractMetadata :: Pandoc -> Either ZettelParseError (Maybe Meta)
 extractMetadata doc
   | Just ((_, _, Map.fromList -> properties), _) <- getH1 doc = do
-    date <- traverse parseDate $ lookup "date" properties
+    createdWithTime <- traverse parseDate $ lookup "created" properties
+    createdDate <- traverse parseDate $ lookup "date" properties
+    let created =
+          case createdWithTime of
+            Nothing -> createdDate
+            _ -> createdWithTime
+
```

Comment:
Refactor this using `<|>`. cf. https://www.parsonsmatt.org/2016/11/18/clean_alternatives_with_maybet.html

Reply:
oh my... I was searching for exacty that, cool